src/elements/content-explorer/ItemGrid.js

```diff
@@ -21,13 +27,33 @@ const ItemGrid = ({ currentCollection, rootId, ...rest }: Props) => {
     const slotRenderer = (slotIndex: number): ?React.Element<any> => {
         const item: ?BoxItem = getProp(currentCollection, `items[${slotIndex}]`);
 
-        return item ? <ItemGridCell item={item} rootId={rootId} {...rest} /> : null;
+        return item ? <ItemGridCell item={item} onItemSelect={onItemSelect} rootId={rootId} {...rest} /> : null;
+    };
+
+    /**
+     * Calls onGridViewResize with the max number of columns that can be displayed given
+     * the current width of the GridView.
+     *
+     * @param {height: number, width: number} dimensions - current dimensions of GridView, as given by AutoSizer
+     * @return {void}
+     */
+    const onResize = ({ width }): void => {
```

Comment:
I wonder if this logic could be somehow reconciled with the `makeResponse` on the `ContentExplorer` component. There we determine whether the size is small, med, or large and pass that as a prop to `ContentExplorer`. Could we somehow utilize that to determine the maxColumns instead having this duplication?